### ISSUE TITLE
 correct ASAP and before June 30 to safely retain the package on CRAN

### ISSUE THREAD

**Issue by cboettig** [ropensci/codemetar#296]
> Package V8 no longer installs on fedora-clang after upgrading to Fedora 32.
> suggest it or a package which requires it but do not use it
conditionally (see §1.1.3.1 of 'Writing R Extensions').
> Please correct ASAP and before June 30 to safely retain the package on CRAN.

**Comment by cboettig** [comment]
I think we have deprecated the jsonld functions and so can drop it from SUGGESTS

**Comment by maelle** [comment]
Could you please edit https://github.com/ropensci/codemetar/blob/00d5fc2fefe921a28a97a880eb8b5dbdf9556425/R/write_codemeta.R#L76?

I'll have little time in the next few weeks myself 😢

**Comment by elinw** [comment]
Did this happen? I just tried to reinstall now and the rstudio mirror said no version for R 4.0.2.

**Comment by maelle** [comment]
Yep see #289 😢 

I hope you can use the GitHub version.

**Comment by RLumSK** [comment]
I bumped into the same problem, so I had a look into it:

1. The packages 'jsonld' and 'V8' are back on CRAN, so theoretically 'codemetar' should make it back on CRAN as well; it would just need a resubmit. 
2. I had a look into the potential consequences of dropping 'jsonld' to prepare a pull request. Unfortunately, I lack the background discussion to that intention. 'jsonld' is used in: 
2.1 `crosswalk.R`
2.2 `codemetar-package.R`
2.3 `codemeta_validate.R`
2.4 `write_codemeta.R`
Is it really the intention to remove all the related code or do I overlook something? On top of it: I did not find 'deprecated' in the code that would suggest that, e.g., the argument ` write_minimeta` in `write_codemeta.R` is of no use anymore. But also here, maybe I am overlooking something.  

Maybe the related arguments should be first 'deprecated' and then dropped with another release?
